ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -230,7 +230,7 @@ public RouterConfig(VerifiableProperties verifiableProperties) {
     routerScalingUnitMaxConnectionsPerPortPlainText =
         verifiableProperties.getIntInRange("router.scaling.unit.max.connections.per.port.plain.text", 5, 1, 20);
```

Comment:
while at it, increase the limit for this too?

Reply:
yes.